### ISSUE TITLE
Wrong RADV/mesa version shown in Steam games

### ISSUE THREAD

**Issue by Mershl** [flightlessmango/MangoHud#564]
### Tested with
- `mangohud-0.6.4-1.fc34.x86_64`
- `steam-1.0.0.70-3.fc34.i686` == `Steam 1623193068`
- `Proton experimental-6.3-20210709`
- `mesa-21.1.3.fc34.x86_64`

### Description
Mangohud shows an unexpected used version of mesa/RADV. Instead of the natively installed version `21.1.3` instead `21.1.4` is shown in Mangohud. Tested with Horizon Zero Dawn (DX12, VKD3D-2.4) on Steam.

Proton log confirms the used Vulkan driver + the log does not state `21.1.4` once:
```
info:  AMD RADV SIENNA_CICHLID (ACO):
info:    Driver: 21.1.3
info:    Vulkan: 1.2.168
```

![image](https://user-images.githubusercontent.com/520251/125137612-557cd180-e10d-11eb-90e8-9edca7d98491.png)

### My guess is either
1) `21.1.4` is the current version of flatpak mesa on my system. As I use the native Steam client I do not expect the flatpak version of mesa to be used. Maybe mangohud is misdirected to flatpak mesa's version string?
2) The version string of `mesa-21.1.3.fc34.x86_64` is just wrong somewhere?
3) ... ?

### Installed mesa packages
```
$ rpm -qa "*mesa*"
mesa-filesystem-21.1.3-1.fc34.x86_64
mesa-libgbm-21.1.3-1.fc34.x86_64
mesa-libglapi-21.1.3-1.fc34.x86_64
mesa-libEGL-21.1.3-1.fc34.x86_64
mesa-libxatracker-21.1.3-1.fc34.x86_64
mesa-vulkan-drivers-21.1.3-1.fc34.x86_64
mesa-dri-drivers-21.1.3-1.fc34.x86_64
mesa-libGL-21.1.3-1.fc34.x86_64
mesa-filesystem-21.1.3-1.fc34.i686
mesa-libGLU-9.0.1-4.fc34.x86_64
mesa-libOSMesa-21.1.3-1.fc34.x86_64
mesa-demos-8.4.0-9.20181118git1830dcb.fc34.x86_64
mesa-libEGL-devel-21.1.3-1.fc34.x86_64
mesa-libGL-devel-21.1.3-1.fc34.x86_64
mesa-libGLU-devel-9.0.1-4.fc34.x86_64
mesa-vulkan-drivers-21.1.3-1.fc34.i686
mesa-libglapi-21.1.3-1.fc34.i686
mesa-libGL-21.1.3-1.fc34.i686
mesa-dri-drivers-21.1.3-1.fc34.i686
```

**Comment by flgx16** [comment]
This should be fixed in MangoHud v0.6.5.

**Comment by Mershl** [comment]
Thanks for the lightning fast response! @flgx16 

https://bodhi.fedoraproject.org/updates/FEDORA-2021-d007730c29
MangoHud v0.6.5 will arrive later today in Fedora 34 stable. I'll confirm & close the issue once it hits.

**Comment by Mershl** [comment]
Can confirm. Fixed with MangoHud v0.6.5.
